msteams-platform/task-modules-and-cards/cards/people-picker-adaptive-card.md

```diff
@@ -0,0 +1,209 @@
+
+**To reassign any task**
+
+1. Select **Reassign** where the people picker field is prepopulated with name to reassign the task to the intended user.
+1. Remove the incorrect user's name. 
+1. Select intended users as per the image scenario, user B (Mona), and user C (Robin) for the task. 
+1.  Select **Assign**. After assigning, the information is sent to the bot. 
+   The bot updates Adaptive Card and notifies the intended users. 
+ 
+The following image shows the reassignment scenario:    
+
+![People Picker on Desktop](../../assets/images/cards/desktoppp.gif)
+
+---
+
+## Implement People Picker
+
+People Picker is implemented as an extension of the [Input.ChoiceSet](https://adaptivecards.io/explorer/Input.ChoiceSet.html) control. The input control includes the following selections:   
+
+* Dropdown, such as an expanded selection.
+* Radio button, such as a single selection.
+* Check boxes, such as multiple selections.  
+
+> [!NOTE]
+> The `Input.ChoiceSet` control is based on the `style` and `isMultiSelect` properties.  
+
+### Update schema
+
+The following properties are the new additions to the `Input.ChoiceSet` schema to enable People Picker experience on the card.  
+
+#### Input.ChoiceSet control
+
+|Property |Type |Required |Description |
+|----|----|----|----|
+|**choices.data** |**Data.Query** |No |Enables dynamic auto complete for different user types, by fetching results from the dataset specified. |
+
+#### Data.Query
+
+|Property |Type |Required |Description |
+|--|--|--|--|
+|**dataset** |String |Yes |The type of data that must be fetched dynamically.|   
+
+#### dataset
+The following table provides predefined values as **dataset** for people picker:   
+
+|dataset|Search Scope
+|--|--|
+|**graph.microsoft.com/users** |Search all members across the organization.|
+|**graph.microsoft.com/users?scope=currentContext** |Search within the members of the current conversation, such as chat or channel in which the particular card is sent.|  
+
+<br> 
+
+> [!NOTE]
+> Currently, the search functionality for all the members across the organization is supported in 1:1 chats with bots, Adaptive Card task modules and tabs only. It is not supported in other 1:1 chats, group chats or channels.  
+
+### Example
+The code example for creating People Picker with organization search is as follows:
+
+```json 
+{
+    "type": "AdaptiveCard",
+    "body": [
+        {
+            "type": "TextBlock",
+            "size": "Medium",
+            "weight": "Bolder",
+            "text": "People Picker with Org search enabled"
+        },
+        {
+            "type": "Input.ChoiceSet",
+            "choices": [],
+            "choices.data": {
+                "type": "Data.Query",
+                "dataset": "graph.microsoft.com/users"
+            },
+            "id": "people-picker",
+            "isMultiSelect": true
+        },
+        {
+            "type": "ActionSet",
+            "actions": [
```

Comment:
Just moving this outside to actions.  Could you please replace this code snippet with following?

```
{
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "size": "Medium",
      "weight": "Bolder",
      "text": "People Picker with Org search enabled"
    },
    {
      "type": "Input.ChoiceSet",
      "choices": [],
      "choices.data": {
        "type": "Data.Query",
        "dataset": "graph.microsoft.com/users"
      },
      "id": "people-picker",
      "isMultiSelect": true
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "Submit"
    }
  ],
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2"
}
```

Reply:
Updated.